spec/features/1_basic_spec.rb

```diff
@@ -0,0 +1,91 @@
+require "rails_helper"
+
+describe "The /movies page" do
+  before do
+    visit "/movies"
+  end
+
+  it "can be visited", points: 1 do
+    expect(page.status_code).to be(200),
+      "Expected to visit /movies successfully."
+  end
+
+  it "has a link to add a movie", points: 1 do
+    expect(page).to have_link('Add a new movie', href: "/movies/new"),
+      "Expected /movies to have an 'Add a new movie' link to '/movies/new'."
+  end
+end
+
+describe "The /movies/new page" do
+  before do
+    visit "/movies/new"
+  end
+
+  it "can be visited", points: 1 do
+    expect(page.status_code).to be(200),
+      "Expected to visit /movies/new successfully."
+  end
+
+  it "has a form", points: 1 do
+    expect(page).to have_selector("form[action='/movies']"),
+      "Expected /movies/new to have a form with action='/movies'."
+  end
+
+  it "has a hidden authenticity token input", points: 2 do
+    expect(page).to have_selector("input[name='authenticity_token']", visible: false),
+      "Expected the new movie form to have an input field of type='hidden' and name='authenticity_token'."
+  end
```

Comment:
Testing this again using the "solutions" from helper-methods-3, I don't think this tag is visible from the `form_with` method.

This always seems to fail for me, even when enabling JavaScript in the test (in case the input is injected later on)

Reply:
@jelaniwoods See my comment about this on the other PR. I know it's weird, but it's actually kind of a good thing that this test doesn't pass with `form_with` here. We want them to write it out by hand and in the next project `helper-methods-3` they get to use `form_with` and we drop the authenticity token test.

Reply:
@bpurinton I thought the ending point of helper-methods is the _starting_ point for helper-methods-3, which uses `form_with`. The current [README for this project says to use `form_with`](https://github.com/appdev-projects/helper-methods#forms-in-general). Is the idea for this assignment that they _also_ include the `authenticity_token` helper even though it shouldn't be required?

Reply:
@jelaniwoods  Ah, you're right. The video recording for this project starts with the `form_with` refactoring. Removing this spec here then.